backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py

```diff
@@ -734,17 +765,34 @@ def _fetch_response(
         x2text = str(profile_manager.x2text.id)
         if not profile_manager:
             raise DefaultProfileError()
-        index_result = PromptStudioHelper.dynamic_indexer(
-            profile_manager=profile_manager,
-            file_path=doc_path,
-            tool_id=str(tool.tool_id),
-            org_id=org_id,
-            document_id=document_id,
-            is_summary=tool.summarize_as_source,
-            run_id=run_id,
-            user_id=user_id,
-            process_text=process_text,
-        )
+        if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
```

Comment:
Trivial suggestion. Looks like there are places where we have handled  as below:
if not FEATURE_FLAG:
else:
Would be good and helpful to keep this consistent.

Reply:
Addressed